src/plugin.js

```diff
@@ -336,7 +336,10 @@ const plugin = function(options) {
       }
 
       if (referenceChild) {
-        const controlBarChild = this.controlBar.addChild('overlay', mergeOptions);
+        const referenceChildIndex = this.controlBar.children().findIndex(function(child) {
```

Comment:
looks like `findIndex` isn't available on IE11 and may break things.

Reply:
I think we can do `this.controlBar.children().indexOf(referenceChild)` directly here.